bitcoin/src/psbt/mod.rs

```diff
@@ -2113,6 +2114,16 @@ mod tests {
         assert_eq!(psbt1, psbt2);
     }
 
+
+    // https://github.com/rust-bitcoin/rust-bitcoin/issues/3628
```

Comment:
It's kind late since this is merged, but why is this issue link here?

Reply:
Just incase people want to reproduce the test files from the fuzz input.

Reply:
I could have been more explicit why I was linking that here.

Reply:
I think that a link to an issue immediately before a testcase is very clear. Though just having 3628 in the name was probably sufficient.